runtime/Cpp/runtime/src/support/Any.cpp

```diff
@@ -0,0 +1,10 @@
+#include "Any.h"
+
+antlrcpp::Any::~Any()
+{
+    delete _ptr;
+}
+
+antlrcpp::Any::Base::~Base()
+{
+}
```

Comment:
I'm usually fine with having the implementation in a cpp file, so that change is ok. But when you start such a clean up it should be done fully, i.e. move all definitions here from the header.